tests/conftest.py

```diff
@@ -504,6 +504,10 @@ def check_shortcut_signature(
     # shortcuts to return more specific types than the bot method, but it's only annotations after
     # all
     for kwarg in effective_shortcut_args:
+        expected_kind = shortcut_sig.parameters[kwarg].kind
+        if bot_sig.parameters[kwarg].kind != expected_kind:
```

Comment:
```suggestion
        expected_kind = bot_sig.parameters[kwarg].kind
        if shortcut_sig.parameters[kwarg].kind != expected_kind:
```
shouldn't this be the other way around? Bot signature should be the 'correct' one, not the other way around

Reply:
Eagle eyes as usual :)